### DISCUSSION TITLE
Lazyvim modifies the entire file in a git repository and not only changed lines

### DISCUSSION THREAD

**Discussion by ziriuz84** [LazyVim/LazyVim#4186] [Q&A]
I'm using lazyvim as daily ide to develop php projects. Occasionally when I save some files git diff shows the entire file as modified, other only empty lines, independently from if I turned on lang.php or not. If I save the same file with VSCode git diff shows only the modified lines. Until a week ago lazyvim worked flawlessly. Something changed in the meantime?

**Comment by folke** [comment]
Nope. Did you make any changes to crlf git setting?
